index.html

```diff
@@ -125,7 +123,7 @@ <h3>Profesional Art Printing Data</h3>
                     </ul>
                 </div>
                 <div class="card-body-bottom">
-                <button class="project-button" type="button"> See Project</button>
```

Comment:
- [ optional] since the previous reviewer didn't mention this it is marked optional. please make the first see project button to be functional, or add a popup to it if you haven't done so. 
![Screenshot (144)_LI](https://user-images.githubusercontent.com/65298048/174076500-97fd722d-7f30-4514-adf7-87e7da10c829.jpg)
- [ optional] please fix the bottom margin of your popup so be clear and visible in the desktop version. the see live and see source buttons ain't visible. 
![Screenshot (145)_LI](https://user-images.githubusercontent.com/65298048/174076839-459ca6eb-5d4a-4c4f-9c60-1a3efc0ed901.jpg)

Reply:
Thank you Tufoin, I didn't that popup needs to be added on the above button too, i have added it now and also fixed the other issue